<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="searchPropertiesGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>11</y>
      <width>840</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>840</width>
      <height>230</height>
     </size>
    </property>
    <property name="title">
     <string>Search Properties</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="flagsMatPropHorizontalLayout">
       <item>
        <widget class="QFrame" name="flagsFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="flagsContainer" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>371</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>371</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="flagsHorizontalLayout">
            <item>
             <widget class="QLabel" name="flagsLabel">
              <property name="text">
               <string>Flags &amp; </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="flags1LineEdit"/>
            </item>
            <item>
             <widget class="QComboBox" name="flagsOpComboBox">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="flags2LineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QCheckBox" name="flagsCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Flags</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="matPropertyFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="matPropCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>141</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Material Property</string>
          </property>
         </widget>
         <widget class="QWidget" name="matPropContainer" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="nuLabel">
              <property name="text">
               <string>NU_</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="matPropLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dstSrcHorizontalLayout">
       <item>
        <widget class="QFrame" name="dstFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="dstCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>dst Factor</string>
          </property>
         </widget>
         <widget class="QWidget" name="dstContainer" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="dstLabel">
              <property name="text">
               <string>dstFactor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dstOpComboBox">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="dstLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="srcFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="srcContainer" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="srcLabel">
              <property name="text">
               <string>srcFactor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="srcOpComboBox">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="srcLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QCheckBox" name="srcCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>src Factor</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="texCountCullModeHorizontalLayout">
       <item>
        <widget class="QFrame" name="cullModeFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="cullModeCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Cull Mode</string>
          </property>
         </widget>
         <widget class="QWidget" name="cullModeContainer" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="cullModeLabel">
              <property name="text">
               <string>Cull Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cullModeOpComboBox">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cullModeLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="textureCountFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="texCountCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Texture Count</string>
          </property>
         </widget>
         <widget class="QWidget" name="dstContainer_2" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_7">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>241</width>
            <height>33</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="texCountLabel">
             <property name="text">
              <string>Count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="texCountOpComboBox">
             <item>
              <property name="text">
               <string>==</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;=</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;=</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="texCountLineEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="alphaFuncTestHorizontalLayout">
       <item>
        <widget class="QFrame" name="alphaFuncFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="alphaFuncCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Alpha Function</string>
          </property>
         </widget>
         <widget class="QWidget" name="dstContainer_4" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_8">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="alphaFunctionLabel">
              <property name="text">
               <string>Function</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="alphaFuncOpComboBox_2">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="alphaFuncLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="alphaTestFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="alphaTestCheckBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Alpha Test</string>
          </property>
         </widget>
         <widget class="QWidget" name="dstContainer_5" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="alphaFunctionLabel_2">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="alphaTestOpComboBox">
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&lt;=</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="alphaTestLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="displayOptionsGroupBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>351</width>
      <height>190</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Display Settings</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="displayFileNameCheckBox">
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displaySrcCheckBox">
       <property name="text">
        <string>Src Factor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayDstCheckBox">
       <property name="text">
        <string>Dst Factor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayFlagsCheckBox">
       <property name="text">
        <string>Flags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayCullCheckBox">
       <property name="text">
        <string>Cull Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayTexturesCheckBox">
       <property name="text">
        <string>Textures</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="noPropertyRadioButton">
         <property name="text">
          <string>No Properties</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="selectedPropertyRadioButton">
         <property name="text">
          <string>Selected Property</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allPropertiesRadioButton">
         <property name="text">
          <string>All Properties</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
